.code-samples.meilisearch.yaml

```diff
@@ -910,25 +910,26 @@ settings_guide_typo_tolerance_1: |-
     }'
 updating_guide_check_version_new_authorization_header: |-
   curl \
-    -X GET 'http://localhost:7700/version' \
+    -X GET 'http://<your-domain-name>/version' \
     -H 'Authorization: Bearer API_KEY'
 updating_guide_check_version_old_authorization_header: |-
   curl \
-    -X GET 'http://localhost:7700/version' \
+    -X GET 'http://<your-domain-name>/version' \
     -H 'X-Meili-API-Key: API_KEY'  
 updating_guide_get_displayed_attributes_new: |-
   # whenever you see {index_uid}, replace it with your index's unique id
   curl \
-    -X GET 'http://localhost:7700/indexes/{index_uid}/settings/displayed-attributes' \
-    -H 'Authorization: Bearer API_KEY'
+    -X GET 'http://<your-domain-name>/indexes/{index_uid}/settings/displayed-attributes' \
+    -H 'X-Meili-API-Key: API_KEY'
 updating_guide_reset_displayed_attributes_new: |-
   curl \
-    -X DELETE 'http://localhost:7700/indexes/{index_uid}/settings/displayed-attributes' \
-    -H 'Authorization: Bearer API_KEY'
+    -X DELETE 'http://<your-domain-name>/indexes/{index_uid}/settings/displayed-attributes' \
+    -H 'X-Meili-API-Key: API_KEY'
```

Comment:
There's a reason these two code samples have "new" at the end: they're supposed to be using the new authorization header. Please rename them accordingly, and don't forget the `sample-template.yaml`.